deepinterpolation/cli/inference.py

```diff
@@ -1,55 +1,11 @@
 import argschema
 import json
-import h5py
-import numpy as np
 from pathlib import Path
 
 from deepinterpolation.cli.schemas import InferenceInputSchema
 from deepinterpolation.generic import ClassLoader
 
 
-def normalize_uint16_output(input_file: Path,
-                            output_file: Path) -> np.ndarray:
-    """function to globally normalize and convert datatype of output
-    movies. The output movie is shifted to be >=0 and scaled to the
-    same max value as the input movie. The movie is zero-padded at the
-    end to match the number of frames of the input movie.
-
-    Parameters
-    ----------
-    input_file: Path
-        the pre-denoising movie. Used to establish histogram bounds
-        of intensity.
-    output_file: Path
-        the post-denoising movie. It is floating point and needs to
-        be scaled and type converted
-
-    Returns
-    -------
-    data: np.ndarray
-        the scaled and type converted result
-
-    Notes
-    -----
-    We anticipate removing this when the Inference class is upgraded to
-    take care of negative values and convert to uint16.
-
-    """
-    with h5py.File(input_file, "r") as f:
-        inmax = f["data"][()].max()
-        inshape = f["data"].shape
-
-    with h5py.File(output_file, "r") as f:
-        out = f["data"][()].squeeze()
-
-    out = (out - out.min()) * inmax / out.ptp()
```

Comment:
I am a very deep conceptual problem with this implementation. Absolute fluorescence values have a meaning. They are  linearly related to photon flux values. By applying a gain to them, we are essentially breaking the data distribution as they are supposed to be. I removed this code because this is conceptually wrong in my opinion. Happy to discuss more.

Reply:
This gain would make it impossible to calculate proper DF/F values.